### DISCUSSION TITLE
Cmake build to VS Code in the New Project file

### DISCUSSION THREAD

**Discussion by sureshsathish1994** [microsoft/vscode-discussions#485] [Extension Show and Tell]
Hello,
 I am new to Cmake to build the projects in VS Code.
 In general - A normal build takes some of the Steps( Create a new project with the Specified device in VS Code ( Json) Build process will handle automatically. )

 Cmake:-
 After creating a new project, then what is the correct procedure that includes Cmake to  Build the project?

 Please mention step by step procedure. 

![TestMainScreenshot](https://user-images.githubusercontent.com/68508461/220633631-19b8de84-67ad-4ba9-81f1-9c009951e488.png)

**Comment by alefragnani** [comment]
Hi @sureshsathish1994 ,

I’m not sure which CMake extension you are using, but this question seems better suited to be asked in its repository. 

This discussion platform is currently focused on developers who are _building extension_ for VS Code.

Hope this helps
